src/patternfly/components/Slider/slider.scss

```diff
@@ -0,0 +1,203 @@
+  --pf-c-slider__step-tick--Height: #{pf-size-prem(4px)};
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--BorderColor--200);
+  --pf-c-slider__step-tick--TranslateX: -50%;
+  --pf-c-slider__step--m-active__slider-tick--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-slider__step--first-child__step-tick--TranslateX: 0;
+  --pf-c-slider__step--last-child__step-tick--TranslateX: -100%;
+
+  // step label
+  --pf-c-slider__step-label--TranslateX: -50%;
+  --pf-c-slider__step-label--Top: calc(var(--pf-global--spacer--xl) + var(--pf-c-slider__rail-track--Height));
+  --pf-c-slider__step--first-child__step-label--TranslateX: 0;
+  --pf-c-slider__step--last-child__step-label--TranslateX: -100%;
+
+  // thumb
+  --pf-c-slider__thumb--Top: calc(var(--pf-c-slider__rail-track--Height) / 2 + var(--pf-global--spacer--md));
+  --pf-c-slider__thumb--Width: #{pf-size-prem(16px)};
+  --pf-c-slider__thumb--Height: #{pf-size-prem(16px)};
+  --pf-c-slider__thumb--Left: var(--pf-c-slider--value);
+  --pf-c-slider__thumb--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-slider__thumb--TranslateX: -50%;
+  --pf-c-slider__thumb--TranslateY: -50%;
+  --pf-c-slider__thumb--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-slider__thumb--BoxShadow--base:
+    0 0 0 2px #fff,
+    0 0 0 3px var(--pf-global--active-color--100);
+  --pf-c-slider__thumb--hover--BoxShadow: var(--pf-c-slider__thumb--BoxShadow--base);
+  --pf-c-slider__thumb--focus--BoxShadow: var(--pf-c-slider__thumb--BoxShadow--base);
+  --pf-c-slider__thumb--active--BoxShadow:
+    var(--pf-c-slider__thumb--BoxShadow--base),
+    0 0 2px 5px var(--pf-global--active-color--200);
+
+  // value
+  --pf-c-slider__value--MarginLeft: var(--pf-global--spacer--md);
+  --pf-c-slider__value--c-form-control--width-base: 3.5ch;
+  --pf-c-slider__value--c-form-control--width-chars: 2;
+  --pf-c-slider__value--c-form-control--Width: calc(var(--pf-c-slider__value--c-form-control--width-base) + (var(--pf-c-slider__value--c-form-control--width-chars) * 1ch));
+  --pf-c-slider__value--m-floating--TranslateX: -50%;
+  --pf-c-slider__value--m-floating--TranslateY: -100%;
+  --pf-c-slider__value--m-floating--Left: var(--pf-c-slider--value);
+  --pf-c-slider__value--m-floating--ZIndex: var(--pf-global--ZIndex--sm);
+
+  // controls
+  --pf-c-slider__controls--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-slider__controls--main--MarginLeft: var(--pf-global--spacer--sm);
+
+  display: flex;
+}
+
+.pf-c-slider__main {
+  position: relative;
+  flex-grow: 1;
+}
+
+.pf-c-slider__rail {
+  padding-top: var(--pf-c-slider__rail--PaddingTop);
+  padding-bottom: var(--pf-c-slider__rail--PaddingBottom);
+}
+
+.pf-c-slider__rail-track {
+  position: relative;
+  height: var(--pf-c-slider__rail-track--Height);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    background:
```

Comment:
Beautiful! 🤩

Reply:
Thanks! And this should still work when we upgraydd the slider to select a range, we'll just introduce more stops.